lib/linguist/languages.yml

```diff
@@ -3753,6 +3753,19 @@ RMarkdown:
   - ".rmd"
   tm_scope: source.gfm
   language_id: 313
+RPC:
+  type: programming
+  aliases:
+  - rpcgen
+  - oncrpc
+  - xdr
+  ace_mode: c_cpp
+  extensions:
+  - ".x"
+  interpreters:
+  - rpcgen
+  tm_scope: source.c
```

Comment:
I found a [grammar for RPC on GitHub](https://github.com/shakeelmohamed/sublime-rpcgen). Could you check if it does a better job than the C grammar, especially regarding the `%` lines mentioned previously? You can use [Lightshow](https://github-lightshow.herokuapp.com/) to test it without installing Sublime Text.

Reply:
It looks pretty much just like C, and doesn't seem to add much useful (like `program` or `string` keywords or special `%` processing). I'd say it's probably not worth it.

Reply:
Ok. Thanks for checking.